pull request

Zerglrisk opened the pull request.

Zerglrisk committed:
GetterSetter, Dowhile

Zerglrisk committed:
add fullname trigger

tested by sublime text 3 3126

Zerglrisk committed:
add Enum

Zerglrisk committed:
Fix Path

etic commented on Code/Enum.sublime-snippet:
```diff
@@ -0,0 +1,10 @@
+{
+	${0}
+}]]></content>
+	<tabTrigger>struct</tabTrigger>
```
Are the tab trigger and description supposed to be enum?

Zerglrisk committed:
fix code

Declare an ${2:VariableName} property of type ${1:DataType}:
to
Declare an ${3:PropertyName} property of type ${1:DataType}:

Zerglrisk committed:
add comment

# How to Insert Not Applied Snippet

paste sublime-snippet file to %appdata%\Sublime Text 3\Packages\User\snippet  folder.

Zerglrisk committed:
fix code

Zerglrisk committed:
Update VariableGetterSetter.sublime-snippet

Zerglrisk committed:
fix
